knip.json

```diff
@@ -0,0 +1,27 @@
+{
+  "workspaces": {
+    "packages/*": {
+      "entry": ["src/index.ts", "test/*.ts"],
+      "project": "**/*.{ts,tsx}"
+    },
+    "apps/*": {
+      "entry": ["src/main.{ts,tsx}", "pages/index.tsx", "test/*.ts"],
```

Comment:
how do we make sure to not forget to add these when we have new apps?

Reply:
`knip` will report unused dependencies if it can't find an entry point. In that case, maybe we need to add the missing entry point or change the `knip` config to support the entry point for that app/package.